include/cantera/thermo/PengRobinson.h

```diff
@@ -205,6 +205,22 @@ class PengRobinson : public MixtureFugacityTP
 
 public:
 
+    //! Returns the isothermal compressibility (\f$\beta_T\f$). Units: 1/Pa.
+    /*!
+     *  \f[ 
+     *    \beta_T = -\frac{1}{v} \left(\frac{\partial v}{\partial p}\right)_T
+     *  \f]
+     */
+    virtual double isothermalCompressibility() const;
```

Comment:
There's no need to repeat documentation for overrides unless there's some specific differences in the documentation, which isn't the case here.

Reply:
I didn't realize the equations were already documented in `ThermoPhase`! The unnecessary comments are now removed.